src/panels/welcome/welcome.js

```diff
@@ -19,6 +21,29 @@ export function initializePanel(panelType, data) {
   
   // Event listener for analyze button
   analyzeBtn.addEventListener('click', async () => {
-    await extensionManager.openPanel('results');
+    // For now, just show a simple message within the button.
+    analyzeBtn.disabled = true;
+    analyzeBtn.textContent = 'Test: Making request and logging to console...';
```

Comment:
I know this is temporary, but we should probably make the messaging a little more finalized for the user. How about: "Analyzing page..."